src/include/misc.h

```diff
@@ -30,6 +30,8 @@
 
 #define	WT_MINUTE	(60)
 
+#define	WT_MSG_PERIOD	(20)
```

Comment:
This is too imprecise `WT_PROGRESS_MSG_PERIOD` would be better. You should also switch the checkpoint progress code to use the define.